views/blocks/career.twig

```diff
@@ -0,0 +1,53 @@
+{% if attributes.isEmpty %}
```

Comment:
instead of storing in a dedicated attribute, you can just test for `attributes.title`. your `modalCanSave` method will ensure they have all the necessary data. oh and do:
```twig
{% if not attributes.title %} 
  <li>stuff you render</li>
{% endif %}
```
placeholder text should be moved to careers.twig

Reply:
Thanks @spelkey-ucd, I've adjusted things from your comments and pushed the latest code.